dart/lib/src/hub.dart

```diff
@@ -149,10 +145,10 @@ class Hub {
 
   /// Captures the message.
   Future<SentryId> captureMessage(
-    String message, {
+    String? message, {
```

Comment:
here it does make sense to be nullable because values come at runtime from external sources

Reply:
it's resolved but still nullable, why would you think it'd be better nullable?

Reply:
maybe a reason would be that people can captureMessage at runtime but not checking if its a null value?